qa-tests-backend/src/test/groovy/ComplianceTest.groovy

```diff
@@ -394,105 +394,101 @@ class ComplianceTest extends BaseSpecification {
 
         then:
         "parse and verify export file"
-        try {
```

Comment:
I don't understand what was tests in this test it's evergreen. All exceptions are catched

Reply:
Is there any history on the try/catch lines to suggest why it was ignoring failures? Turning it back on seems like a good idea but if the test was a WIP then it might be more headache than it is worth.

Reply:
Unfortunately it's the only usage of this function :(
I think we can keep this test

Reply:
@gavin-stackrox I reverted this change and will create a separate PR for that. Now only change println